vanilla/scss/styles.scss

```diff
@@ -70,3 +70,45 @@ li > p {
 .admonition-security-notice {
   padding-left: 1rem;
 }
+
+// Toc custom styles
+.p-table-of-contents__nav > ul > li:first-child > a {
+  display: none;
+}
+
+.p-table-of-contents__nav > ul {
+  @extend .p-list;
+}
+
+.p-table-of-contents__nav > ul > li > ul {
+  margin-left: 0;
+  padding-left: 0;
+}
+
+.reference.internal {
+  @extend .p-table-of-contents__link;
+}
+
+.is-sticky {
+  max-height: 100vh;
+  overflow-y: auto;
+  position: sticky;
+  top: 0;
+}
+
+.headerlink {
+  display: none;
+}
+
+h1:is(:focus, :hover),
```

Comment:
I didn't know that notation. Would `h1:focus, h1:hover, …` work as well? It seems it would be more robust (but a bit more repetitive).

Reply:
I read this this morning 😅 https://www.aleksandrhovhannisyan.com/blog/writing-better-css/ and it was in there so thought I'd give it a shot. If you think it's better to keep it the usual way happy to change it

Reply:
OK, you young people coming with your fancy new CSS features :D Maybe it's time to learn something as well ;)

I guess it makes sense to keep it, especially now that we dropped IE support. It seems to have its benefits.

Reply:
I am a pretty fancy developer to be fair 💅. Ok, we'll keep it!